include/z64vis.h

```diff
@@ -0,0 +1,87 @@
+#ifndef Z64_VIS_H
+#define Z64_VIS_H
+
+#include "ultra64.h"
+#include "color.h"
+
+typedef enum {
+    /* 0 */ FB_FILTER_NONE,
+    /* 1 */ FB_FILTER_CVG_RGB,
+    /* 2 */ FB_FILTER_CVG_RGB_UNIFORM,
+    /* 3 */ FB_FILTER_CVG_ONLY,
+    /* 4 */ FB_FILTER_CVG_RGB_FOG, // Not recommended, easily overflows blender
+    /* 5 */ FB_FILTER_ZBUF_IA,
+    /* 6 */ FB_FILTER_ZBUF_RGBA,
+    /* 7 */ FB_FILTER_MONO
+} FramebufferFilterType;
+
+typedef enum {
+    /* 0 */ VIS_NO_SETSCISSOR,
+    /* 1 */ VIS_SETSCISSOR
+} VisScissorType;
+
+typedef struct {
+    /* 0x00 */ u32 type;
+    /* 0x04 */ u32 setScissor;
+    /* 0x08 */ Color_RGBA8_u32 primColor;
+    /* 0x0C */ Color_RGBA8_u32 envColor;
+} Vis; // size = 0x10
+
+
+
+/* Cvg: Coverage */
+
+#define FB_FILTER_TO_CVG_TYPE(filter) (filter)
+
+typedef enum {
+    /* 0 */ VIS_CVG_TYPE_NONE = FB_FILTER_TO_CVG_TYPE(FB_FILTER_NONE),
+    /* 1 */ VIS_CVG_TYPE_CVG_RGB = FB_FILTER_TO_CVG_TYPE(FB_FILTER_CVG_RGB),
+    /* 2 */ VIS_CVG_TYPE_CVG_RGB_UNIFORM = FB_FILTER_TO_CVG_TYPE(FB_FILTER_CVG_RGB_UNIFORM),
+    /* 3 */ VIS_CVG_TYPE_CVG_ONLY = FB_FILTER_TO_CVG_TYPE(FB_FILTER_CVG_ONLY),
+    /* 4 */ VIS_CVG_TYPE_CVG_RGB_FOG = FB_FILTER_TO_CVG_TYPE(FB_FILTER_CVG_RGB_FOG)
+} VisCvgType;
+
+typedef struct {
+    /* 0x00 */ Vis base;
+} VisCvg; // size = 0x10
```

Comment:
i think just `vis` could be nice instead of base for all of these. its like `Actor actor`, saying what it is, and its a plus that its 1 char shorter lol